src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecution.tsx

```diff
@@ -351,6 +351,7 @@ export const useTradeExecution = (
             stepIndex: hopIndex,
             slippageTolerancePercentageDecimal,
             xpub,
+            senderAddress: _senderAddress,
```

Comment:
Using the version without the bitcoincash: prefix stripped as stripping it is mostly an us concern (and also this is unused anywhere but in chainflip for the purpose of passing refund address in)